keyboards/chocofly/chocofly.c

```diff
@@ -0,0 +1,16 @@
+// Copyright 2022 Vitaly Volkov (@vlkv)
+// SPDX-License-Identifier: GPL-2.0-or-later
+
+#include "chocofly.h"
+
+#ifdef ENCODER_ENABLE
+bool encoder_update_kb(uint8_t index, bool clockwise) {
+    if (!encoder_update_user(index, clockwise)) { return false; }
+    if (clockwise) {
+        tap_code(KC__VOLUP);
+    } else {
+        tap_code(KC__VOLDOWN);
```

Comment:
```suggestion
        tap_code(KC_VOLU);
    } else {
        tap_code(KC_VOLD);
```

Reply:
Thank you, I converted these to the single underscore version.